src/diffusers/pipelines/stable_diffusion/__init__.py

```diff
@@ -1,5 +1,30 @@
 # flake8: noqa
-from ...utils import is_transformers_available
+from dataclasses import dataclass
+from typing import List, Union
+
+import numpy as np
+
+import PIL
+from PIL import Image
+
+from ...utils import ModelOutput, is_transformers_available
+
+
+@dataclass
+class StableDiffusionOutput(ModelOutput):
+    """
+    Output class for image pipelines.
```

Comment:
```suggestion
    Output class for Stable Diffusion pipelines.
```

Distinction made to prevent ambiguity with `ImagePipelineOutput`.